src/Stryker.Core/Stryker.Core/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Visual studio automatically restores packages in a package.json and we already recommend to use visual studio in the contribution guide if I'm not mistaken. Nevertheless it might be better to also document this somewhere. 

With at the root of the project do you mean the root of stryker.core or the solution root? Since it is a compile time dependency of stryker.core specifically I would expect the package json to also be there.

Reply:
I was using Visual Studio 2019 when I pulled the code and I got a build error when trying to run. It definitely did not restore the npm package. Could it be because we run Stryker.CLI and not Stryker.Core?